mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -464,6 +464,34 @@
 
     <string name="error_hostLookup_title">Server not found</string>
 
+    <!-- Onboarding: Title -->
+    <!-- %1$s will be replaced by the app name and build variant(e.g. Firefox Focus, Firefox Klar Nightly) -->
+    <string name="onboarding_title">Welcome to %1$s!</string>
+
+    <!-- Onboarding: Description -->
+    <string name="onboarding_description">Next-level private browsing when you want it</string>
+
+    <!-- Onboarding: Incognito Title -->
+    <string name="onboarding_incognito_title">Where incognito meets invisible</string>
+
+    <!-- Onboarding: Incognito Description -->
+    <string name="onboarding_incognito_description">Focus is a privacy-first browser with extra-strong tracking and cookie protection built in.</string>
```

Comment:
"Focus" should be a variable since it can also be referred to as "Klar"
(and variable should be explained in comments)